src/org/zaproxy/zap/extension/formHandler/OptionsFormHandlerPanel.java

```diff
@@ -0,0 +1,182 @@
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.formHandler;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.SortOrder;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.OptionsParam;
+import org.parosproxy.paros.view.AbstractParamPanel;
+import org.parosproxy.paros.view.View;
+
+import org.zaproxy.zap.view.AbstractMultipleOptionsTablePanel;
+
+public class OptionsFormHandlerPanel extends AbstractParamPanel {
+
+	private static final long serialVersionUID = 1L;
+
+	private FormHandlerMultipleOptionsPanel tokensOptionsPanel;
+
+	private OptionsFormHandlerTableModel FormHandlerModel = null;
+
+    public OptionsFormHandlerPanel() {
+        super();
+ 		initialize();
+    }
+
+	/**
+	 * This method initializes this
+	 */
+	private void initialize() {
+		this.setName(Constant.messages.getString("formHandler.options.title"));
+        this.setLayout(new GridBagLayout());
+
+        GridBagConstraints gbc = new GridBagConstraints();
+        gbc.gridx = 0;
+        gbc.weightx = 1.0;
+        gbc.anchor = GridBagConstraints.LINE_START;
+        gbc.fill = GridBagConstraints.BOTH;
+
+        this.add(new JLabel(Constant.messages.getString("formHandler.options.label.description")), gbc);
+
+        tokensOptionsPanel = new FormHandlerMultipleOptionsPanel(getFormHandlerModel());
+
+        gbc.weighty = 1.0;
+        this.add(tokensOptionsPanel, gbc);
+	}
+
+	@Override
+    public void initParam(Object obj) {
+	    OptionsParam optionsParam = (OptionsParam) obj;
+		FormHandlerParam param = optionsParam.getParamSet(FormHandlerParam.class);
+	    getFormHandlerModel().setTokens(param.getTokens());
+	    tokensOptionsPanel.setRemoveWithoutConfirmation(!param.isConfirmRemoveToken());
+    }
+
+	@Override
+    public void saveParam(Object obj) throws Exception {
+	    OptionsParam optionsParam = (OptionsParam) obj;
+		FormHandlerParam formHandlerParam = optionsParam.getParamSet(FormHandlerParam.class);
+	    formHandlerParam.setTokens(getFormHandlerModel().getElements());
+	    formHandlerParam.setConfirmRemoveToken(!tokensOptionsPanel.isRemoveWithoutConfirmation());
+    }
+
+	/**
+	 * This method initializes FormHandlerTableModel	
+	 * 	
+	 * @return OptionsFormHandlerTableModel	
+	 */    
+	private OptionsFormHandlerTableModel getFormHandlerModel() {
+		if (FormHandlerModel == null) {
+			FormHandlerModel = new OptionsFormHandlerTableModel();
+		}
+		return FormHandlerModel;
+	}
+
+	@Override
+	public String getHelpIndex() {
+		return "ui.dialogs.options.FormHandler";
```

Comment:
It has been updated to the proper target now. However, I've found that the help dialog does not seem to be displayed in my environment upon clicking(For this add on, or others that I know have a working help button). There are no exceptions being thrown in the console either. Any ideas as to what could be causing this?